mobile/cmp/header/AppBar.js

```diff
@@ -114,19 +72,54 @@ export class AppBar extends Component {
                         ...rightItems || [],
                         refreshButton({
                             omit: hideRefreshButton,
-                            disabled: navigatorModel.disableAppRefreshButton,
+                            disabled: navigatorModel?.disableAppRefreshButton,
```

Comment:
Can we get the NavigatorModel from context here? The NavigatorBackButton is doing that, and the two kind of go together

Reply:
good catch -- will merge this, and include that enhancement in my final round of changes tonight.